awx/main/tasks/system.py

```diff
@@ -458,11 +457,11 @@ def inspect_execution_nodes(instance_list):
 
             # Only execution nodes should be dealt with by execution_node_health_check
             if instance.node_type == 'hop':
+                logger.warn(f'Hop node {hostname}, has rejoined the receptor mesh')
```

Comment:
The `.warn()` method is deprecated and throws a warning.  Please use `.warning()` instead.

Reply:
d'oh, I should have known better.